Objective-C/CBLReplicator.mm

```diff
@@ -423,13 +424,21 @@ - (void) setProgressLevel: (CBLReplicatorProgressLevel)level {
 
 - (NSSet<NSString*>*) pendingDocumentIDsForCollection: (CBLCollection *)collection
                                                 error: (NSError**)error {
+    CBLAssertNotNil(collection);
     
-    // TODO: Add implementation
+    CBLStringBytes name(collection.name);
+    CBLStringBytes scopeName(collection.scope.name);
+    C4CollectionSpec c4spec = { .name = name, .scope = scopeName };
     
-    return nil;
+    return [self pendingDocumentIDsForCollectionSpec: c4spec error: error];
 }
 
 - (NSSet<NSString*>*) pendingDocumentIDs: (NSError**)error {
+    return [self pendingDocumentIDsForCollectionSpec: kC4DefaultCollectionSpec error: error];
+}
+
+// internal
+- (NSSet<NSString*>*) pendingDocumentIDsForCollectionSpec: (C4CollectionSpec)spec error: (NSError**)error {
```

Comment:
Why doesn't this get implemented in `- (NSSet<NSString*>*) pendingDocumentIDsForCollection: (CBLCollection *)collection error: (NSError**)error` itself? Then the old pendingDocumentIDs will be called into the new one.

Reply:
it was to avoid getting the default collection but use the default collection spec.